MPU/src/pedals.cpp

```diff
@@ -43,11 +43,27 @@ FaultStatus_t Pedals::readAccel()
 
 	//Calculate the correct amount of torque to command using precentage pressed value
 	appliedTorque = calcTorque(multiplier);
+	float mph = (motorController->getMotorSpeed() * MOTOR_RPM_TO_MPH_CONST);
+
+	if (drive_state == PIT || drive_state == REVERSE) {
+		//Results in a value from 0.5 to 0 (at least halving the max torque at all times in pit or reverse)
+		// float torque_derating_factor = 0.5 + ((-0.5/PIT_MAX_SPEED) * mph);
```

Comment:
why'd you comment my calc out :( I'd prefer you use it over the increment of 10 version you have rn because this is continuous

Reply:
it wasnt working for some reason, we can go back to it if we test it and it ends up working. I like urs more too it just wasnt working and i went with something i was more comfortable with for debugging

Reply:
In my thingy I forgot to add in the ```if motor speed > speed limit torque = 0``` to round off the piecewise function. So I think literally just move that calc into the else